docker-compose.yaml

```diff
@@ -1,26 +1,58 @@
-version: '3.9.1'
+version: "3.9.1"
 
-# You must manually run `ankerctl config import` atleast once
-# docker run \
-# -v ankerctl_vol:/root/.config/ankerctl \
-# -v "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json" \
-# ankerctl config import /tmp/login.json
 services:
-  ankerctl:
-    image: ankerctl/ankerctl:latest
-    container_name: ankerctl
-    restart: unless-stopped
-    build: .
-    environment:
-      - FLASK_PORT=4470
-      - FLASK_HOST=0.0.0.0
-    volumes:
-      - ankerctl_vol:/root/.config/ankerctl
-    ports:
-      - 127.0.0.1:4470:4470
-    entrypoint: "/app/ankerctl.py"
-    command: ["webserver", "run"]
+    ankerctl_seed_mac:
+        image: busybox
+        container_name: ankerctl_seed_mac
+        restart: "no"
+        volumes:
+            - ankerctl_vol:/root/.config/ankerctl
+            - "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json"
+        command: sh -c "cp /tmp/login.json /root/.config/ankerctl/login.json || true"
+    ankerctl_seed_linux:
+        image: busybox
+        container_name: ankerctl_seed_linux
+        restart: "no"
+        volumes:
+            - ankerctl_vol:/root/.config/ankerctl
+            # TODO: verify this is the correct linux path
+            - "$HOME/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json"
+        command: sh -c "cp /tmp/login.json /root/.config/ankerctl/login.json || true"
+    ankerctl_import:
+        # image: ghcr.io/ankermgmt/ankermake-m5-protocol:latest
+        image: ankerctl/ankerctl:latest
+        container_name: ankerctl_import
+        restart: "no"
+        build: .
+        volumes:
+            - ankerctl_vol:/root/.config/ankerctl
+        entrypoint: "/app/ankerctl.py"
+        command: ["config", "import", "/root/.config/ankerctl/login.json"]
+        depends_on:
```

Comment:
No `ankerseed` for Windows? Bind volumes work in Windows too.

Reply:
Isn't this just a naming thing though? It's called ankerctl_import and is still importing the login.json file right?

Reply:
Good point, lets scrap linux support, and add windows support. Do you know the path on windows?

Reply:
The way I understand it, the `ankerctl_seed_*` containers try to copy-in a `login.json` file from the host system, and `ankerctl_import` runs the import action.

@spuder could we make `ankerctl_import` responsible for all the `cp ... || true` attempts, and also `ankerctl.py config import`? That would make this a simple two-step thing.

Reply:
> Good point, lets scrap linux support, and add windows support. Do you know the path on windows?

`%LOCALAPPDATA%\Ankermake\AnkerMake_64bit_fp\login.json`

Reply:
@spuder See above

Reply:
Or `c:\Users\$USER\AppData\Local\Ankermake\AnkerMake_64bit_fp\login.json`

Reply:
I've tried a few local edits and can't find a way of getting thsi to work for Windows using Docker Desktop on WSL.